docs/user-guide/walkthrough/index.md

```diff
@@ -64,11 +53,12 @@ $ kubectl get pods
 
 On most providers, the pod IPs are not externally accessible. The easiest way to test that the pod is working is to create a busybox pod and exec commands on it remotely. See the [command execution documentation](/docs/user-guide/getting-into-containers/) for details.
 
-Provided the pod IP is accessible, you should be able to access its http endpoint with curl on port 80:
+Provided the pod IP is accessible, you should be able to access its http endpoint with wget on port 80:
```

Comment:
The wget is actually to be run in the busybox container (which has wget, but doesn't have curl).  Is there a syntax for representing running the command within the container?  e.g. `pod@busyboxy$ wget -q0- http://$POD_IP`?